app/models/katello/erratum.rb

```diff
@@ -113,7 +113,7 @@ def update_from_json(json)
       if self.updated.blank? || (custom_json['updated'].to_datetime != self.updated.to_datetime)
         custom_json['errata_id'] = custom_json.delete('id')
         custom_json['errata_type'] = custom_json.delete('type')
-
+        custom_json['updated'] = custom_json['updated'].blank? ? custom_json['issued'] : custom_json['updated']
```

Comment:
You might do this above the `if` statement above, that way this will be fixed on reindexing.

Also, would you be able to add a test to test this condition?